common/changes/@rushstack/heft/ianc-remove-custom-fs_2020-09-18-21-03.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "",
+      "type": "none"
```

Comment:
Why is this `"none"`? Seems like a pretty significant change.

Reply:
It's just a refactor. Nothing should change for any users.